src/loaders/svg/tvgSvgCssStyle.cpp

```diff
@@ -175,9 +175,6 @@ void cssUpdateStyle(SvgNode* doc, SvgNode* style)
             if (auto cssNode = cssFindStyleNode(style, nullptr, (*child)->type)) {
                 cssCopyStyleAttr(*child, cssNode);
```

Comment:
It looks like this line is also not used.
If the title is null, nothing is done in `cssFindStyleNode()`, so the entire line 175 to 180 looks unused.
I put the test code(printf()) on lines 176 and 179 along with samples from https://github.com/thorvg/thorvg/issues/1255 and tested it.
And if cssNode is not found here, it is expected to be found in cssApplyStyleToPostponeds().

Reply:
@JSUYA from ```cssFindStyleNode```:
```if ((!title && !(*child)->id) || (title && (*child)->id && !strcmp((*child)->id, title))) return (*child);```

and sample:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200">
 <defs>
  <style>
    rect {
        fill: #00cc00;
        stroke: #99ff33;
    }
  </style>
 </defs>
 <rect x="5" y="5" width="50" height="50" />
</svg>
```
we have in images ```css-style.svg``` sample file.